chamber/models/__init__.py

```diff
@@ -41,52 +43,74 @@ def model_to_dict(instance, fields=None, exclude=None):
     return data
 
 
-class ModelDiffMixin(object):
-    """
-    A model mixin that tracks model fields' values and provide some useful api
-    to know what fields have been changed.
-    """
+ValueChange = collections.namedtuple('ValueChange', 'initial current')
```

Comment:
trochu se mi nelíbí specifikovat ty atributy jedním stringem, sice to jde, ale přijde mi to nestandardní. Radši bych tam měl tuple.